java_tests/src/test/java/org/lsst/sal/SAL_ScriptTest.java

```diff
@@ -0,0 +1,31 @@
+package org.lsst.sal;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+/**
+ * A few tests require CSC that doesn't use generics, so we use Script.
+ */
```

Comment:
The way SAL handles generic topics has changed, making this comment and the test names and details somewhat out of date. The XML field name has changed from Generics to AddedGenerics and "no" is no longer a valid entry. But this is fundamentally a test of the way that SAL handles generic topics, and Script is still unusual in not being a CSC.

Please at least update the code comment, test class name and test name(s) to reflect the current reality.

Please also consider updating the test itself, e.g.:
* Test that `Script` has no `summaryState` event nor `enable` command (two topics that all CSCs have). This is a useful change from testing for the `enterControl` command, because now very CSCs have the `enterControl` command so it is normal for it to be missing.
* Test that `Test` has no `enterControl` command.

Reply:
If you prefer, I can update the Python code first.

Reply:
Please update the Python code first. Unfortunately I am very busy with the dome software installation at the summit and therefore prefer to not have to spend too much time on this ticket if I can possibly help it.

Reply:
I filed DM-32956 to fix the Python and implemented it in https://github.com/lsst-ts/ts_sal/pull/168.

I would rather have this bit of Java cleaned up before merging, even if it adds a delay while you deal with more important things. I am willing to do the cleanup myself, if need be.